src/renderer/components/Index/Inventory.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+.backgrounds_container
+    button.hover-button
+        p Inventory
+    - for (let a=0; a < 10; a++)
+        .invetory-item
+</template>
+
+<style lang="stylus">
+
+$color-1 = #12151a
+$color-2 = #000
+$color-3 = #08090b
+$color-4 = #1c2129
+$color-5 = #fff
+$color-6 = rgba(0,0,0,0.4)
+$color-7 = #36bbf7
+$color-8 = #702df9
+$color-9 = #272d38
+
+.backgrounds_container
+    position absolute
```

Comment:
Try to remove position absolute if possible. I'm not sure if it is, but position absolute not best friend.